src/components/panel/panel.spec.js

```diff
@@ -1055,10 +1056,55 @@ describe('$mdPanel', function() {
         pressEscape();
 
         expect(myButton).toBeFocused();
+
       });
     });
   });
 
+  describe('postClose logic:', function() {
+      it('postClose when provided', function () {
```

Comment:
Follow the same test naming as the other events we fire, like `onOpenComplete` or `onRemoving`.